packages/@markuplint/selector/src/selector.ts

```diff
@@ -468,6 +468,16 @@ class SelectorTarget {
 		}
 	}
 
+	// cspell:disable
+	/**
+	 * Matching is executed in this order: ID > tag name > classes > attributes > psuedo-elements.
```

Comment:
@exoego 

```suggestion
	 * Matching is executed in this order: ID > tag name > classes > attributes > pseudo-elements.
```

And, remove `// cspell:disable`

Reply:
https://github.com/markuplint/markuplint/pull/1485/commits/96bb8974cb2565b879abc2622d85caf19c1549ab